chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -29,28 +47,23 @@ data:
         return 200 "healthy\n";
       }
 
+      # The default cluster running on the same cluster as Kubeapps.
+      location ~* /api/clusters/default {
```

Comment:
so the plan here is to add some templating here to generate blocks like this one and point to different clusters, right?